forem/forem
# Load `post_comments` billboard asynchronously

## What type of PR is this? (check all applicable)


## Description

As a starting point we want to load the `post_comments` billboard on the article#show pages asynchronously. The `post_sidebar` will be done separately.

In addition, we want to ensure that the non-signed in traffic is edge cached even when doing the async loading. 

## Related Tickets & Documents


- Related Issue #https://github.com/forem/forem/issues/19227
- Closes #

## QA Instructions, Screenshots, Recordings

Create a Billboard on the admin to target `post_comments`
Load up an article and ensure that the ad is loading asynchronously. 
You can also open up the network panel to see the html response. 

Full demo can be found here

https://www.loom.com/share/3d937c46682d4ba2ac8cc6c41ac51d0e

### UI accessibility concerns?

No new UI elements added

## Added/updated tests?

      have not been included_

## [optional] Are there any post deployment tasks we need to perform?

We need to monitor the caching, I'll be co-ordinating with Mac.

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/8vQSQ3cNXuDGo/giphy.gif)
